### ISSUE TITLE
How to view the image in collection mode. 

### ISSUE THREAD

**Issue by gaganscanup** [openseadragon/openseadragon#1768]
Actually I have 1280x960 images (let say 20) , I'm using collection mode display with 5 rows, but i'm getting Gap between image to image with all rows.

`     var viewer = OpenSeadragon({
            id:            "openseadragon",
            prefixUrl:     "../images/",
            debugMode:          false,
	    showNavigator:  	true,
	    showRotationControl: true,
            collectionMode:       true,
            collectionRows:       2,
            collectionTileSize:   400,
            collectionTileMargin: -1,
            collectionLayout:     'vertical',
	    visibilityRatio: 1.0,
	    maxZoomPixelRatio: 5,
	    immediateRender: true,
            tileSources: tileSources
        });`

**Comment by iangilman** [comment]
Yes, this is a limitation of the collection mode. More discussion here: https://github.com/openseadragon/openseadragon/issues/1723

**Comment by gaganscanup** [comment]
Okay iangilman
